### PR TITLE
Help popups.

### PR DESCRIPTION
**Pull request by azubieta**
Implements #178 

THIS IS A WORK IN PROGRESS DO NOT MERGE

### CODE REVIEW

File: src/MainWindow.cpp [hunk 1]
```diff
@@ -42,16 +42,16 @@ void MainWindow::initHelpLabels()
     };
 
     ui->label_exportDBHelp->setPixmap(getFontAwesomeIconPixmap(fa::questioncircle));
-    ui->label_exportDBHelp->setToolTip("Placeholder help hint.\n");
+    ui->label_exportDBHelp->setToolTip(tr("Placeholder help hint."));
```
Comment (azubieta) [inline src/MainWindow.cpp]: Change the placeholder text for the real help message.

**Comment by azubieta** [comment]
For the Synchornization tab change the placeholder text for the real help messages [here](https://github.com/mooltipass/moolticute/pull/185/commits/901b13c40d67e238efd1bce6e9011fc26ee9739a#diff-1c0b64226a129d35f02dc0049484cd6bR45)

For the device settings tab use the ui editor and modify the tooltip property of every settings widget item.

**Comment by limpkin** [comment]
@raoulh how would you prefer to proceed for that PR? Would you like me to work on @azubieta's branch or can you merge that one and I'll add the texts just after?

**Comment by limpkin** [comment]
@azubieta I don't see any tooltip for the individual settings... is that normal?

**Comment by azubieta** [comment]
For the individual settings you should modify the tooltip field in the widget settings. This is done using the qt creator UI editor.
For the demonstration of how it's done I just filed the first two settings items. The rest still has the same filed.

**Comment by azubieta** [comment]
For the individual settings you should modify the tooltip field in the widget settings. This is done using the qt creator UI editor.
For the demonstration of how it's done I just filed the first two settings items. The rest still has the same filed.

**Pull request merged**